packages/plugin-infra/src/web3/useFungibleAssets.ts

```diff
@@ -52,39 +52,41 @@ export function useFungibleAssets<
                 const aUSD = toZero(a.value?.[CurrencyType.USD] ?? '0')
                 const zUSD = toZero(z.value?.[CurrencyType.USD] ?? '0')
 
-                const isNativeTokenA = isSameAddress(a.address, Others?.getNativeTokenAddress(a.chainId))
-                const isNativeTokenZ = isSameAddress(z.address, Others?.getNativeTokenAddress(z.chainId))
+                try {
```

Comment:
It's ugly. Any better way to fix it?

Reply:
Maybe `getMaskTokenAddress` doesn't throw an error, only returns `undefined`.